cri/annotations/annotations.go

```diff
@@ -14,6 +14,9 @@ const (
 	// SandboxName is the sandbox name annotation
 	SandboxName = "io.kubernetes.cri-o.SandboxName"
```

Comment:
Actually, I think this shoud be `SanboxID` , but not `SanboxName`.

Reply:
It will use `SandboxName` as sharedContainer for runv, please refer to https://github.com/hyperhq/runv/blob/master/cli/create.go#L96

Reply:
OK, since kata need `SandboxID`, just append it.